libvncclient/vncviewer.c

```diff
@@ -540,7 +547,8 @@ void rfbClientCleanup(rfbClient* client) {
     client->clientData = next;
   }
 
-  free(client->vncRec);
+  if(client->vncRec)
```

Comment:
minor change request: you can leave this as-is as `free(NULL) is defined as NOP`

Reply:
cherry-picked anyway.